packages/sdk/src/health_indicator/health_indicator.ts

```diff
@@ -19,20 +20,32 @@ interface IHealthIndicator {
 }
 
 export class HealthIndicator implements IHealthIndicator {
+  private isStarted = false;
+
   private readonly libp2p: Libp2p;
   private readonly events: IWakuEventEmitter;
 
   private value: HealthStatus = HealthStatus.Unhealthy;
+  private readonly debouncedAssessHealth: ReturnType<typeof debounce>;
```

Comment:
it's clear that it's debounced based on `typeof debounce`, i think keeping it `assessHealth` is better